plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/components/inspector.tsx

```diff
@@ -59,64 +59,78 @@ export const Inspector = ( {
-				<ToggleControl
-					label={ __( 'Display product count', 'woocommerce' ) }
-					checked={ showCounts }
-					onChange={ setCountVisibility }
-				/>
-			</PanelBody>
+		<>
+			<InspectorControls key="inspector">
+				<PanelBody title={ __( 'Display', 'woocommerce' ) }>
+					<ToggleControl
+						label={ __( 'Display product count', 'woocommerce' ) }
+						checked={ showCounts }
+						onChange={ setCountVisibility }
+					/>
+				</PanelBody>
 
-			<PanelBody title={ __( 'Minimum rating', 'woocommerce' ) }>
-				<RadioControl
-					selected={ minRating }
-					className="wc-block-rating-filter__rating-control"
-					options={ [
-						{
-							label: (
-								<MinimumRatingLabel
-									stars={ 4 }
-									ariaLabel={ __(
-										'Four stars and up',
-										'woocommerce'
-									) }
-								/>
-							),
-							value: '4',
-						},
-						{
-							label: (
-								<MinimumRatingLabel
-									stars={ 3 }
-									ariaLabel={ __(
-										'Three stars and up',
-										'woocommerce'
-									) }
-								/>
-							),
-							value: '3',
-						},
-						{
-							label: (
-								<MinimumRatingLabel
-									stars={ 2 }
-									ariaLabel={ __(
-										'Two stars and up',
-										'woocommerce'
-									) }
-								/>
-							),
-							value: '2',
-						},
-						{
-							label: __( 'No limit', 'woocommerce' ),
-							value: '0', // no limit
-						},
-					] }
-					onChange={ setMinRating }
-				/>
-			</PanelBody>
-		</InspectorControls>
+				<PanelBody title={ __( 'Minimum rating', 'woocommerce' ) }>
+					<RadioControl
+						selected={ minRating }
+						className="wc-block-rating-filter__rating-control"
+						options={ [
+							{
+								label: (
+									<MinimumRatingLabel
+										stars={ 4 }
+										ariaLabel={ __(
+											'Four stars and up',
+											'woocommerce'
+										) }
+									/>
+								),
+								value: '4',
+							},
+							{
+								label: (
+									<MinimumRatingLabel
+										stars={ 3 }
+										ariaLabel={ __(
+											'Three stars and up',
+											'woocommerce'
+										) }
+									/>
+								),
+								value: '3',
+							},
+							{
+								label: (
+									<MinimumRatingLabel
+										stars={ 2 }
+										ariaLabel={ __(
+											'Two stars and up',
+											'woocommerce'
+										) }
+									/>
+								),
+								value: '2',
+							},
+							{
+								label: __( 'No limit', 'woocommerce' ),
+								value: '0', // no limit
+							},
+						] }
+						onChange={ setMinRating }
+					/>
+				</PanelBody>
+			</InspectorControls>
+
+			<InspectorControls group="styles">
+				<PanelBody title={ __( 'Display', 'woocommerce' ) }>
+					<ToggleControl
+						label={ __( 'Clear button', 'woocommerce' ) }
+						checked={ clearButton }
+						onChange={ ( value ) =>
+							setAttributes( { clearButton: value } )
+						}
+					/>
+				</PanelBody>
+			</InspectorControls>
+		</>
```

Comment:
This is creating two `Display` panels in two tabs. Looks like we need to review the placement of the Display panel. I prefer to place that panel under the Settings tab over the Styles tab. But that should be addressed in a follow-up PR.